app/controllers/admin/users_controller.rb

```diff
@@ -41,7 +41,13 @@ def verify
 
   def refund_queue
     @title = "Refund queue"
-    @users = User.refund_queue
+
+    render inertia: "Admin/RefundQueues/Show",
+      props: {
+        users: User.refund_queue
```

Comment:
Not including pagination in this PR as it adds a fair bit of complexity, and in theory this shouldn't perform worse than the current UI

Reply:
I noticed `Admin::RefundQueuesController#show` was reverted to a custom `refund_queue` action in `UsersController`. Since we're in the middle of a rewrite, this feels like a missed opportunity to improve the architecture. The extracted controller follows Rails REST conventions, is more maintainable, and treats the refund queue as a proper resource. We're already actively rewriting this area, so now seems like the right time to fix these patterns rather than preserve them.